src/main/resources/application.properties

```diff
@@ -7,7 +7,7 @@ keycloak.ssl-required=external
 #keycloak client that is used for procedures
 keycloak.resource=AI4HF-Auth
 #Chosen keycloak client secret. Should be edited after each Docker compose for the Keycloak to work as intended
-keycloak.credentials.secret=pZlLseohp9n7dOiIwVkHNzswGCeYUH6O
+keycloak.credentials.secret=i9Gm2CbhusqLUGmamK25yvo9ZHf69QIj
```

Comment:
Please revert this change back. You can change it in your local while developing the code.